test/unit/combined_string_field_test.rb

```diff
@@ -78,7 +78,7 @@ module ActiveRecordStub
       begin
         @instance.first = "abc\n123"
       rescue ArgumentError => ex
-        ex.message =~ /Cannot store newlines in combined fields storing \"abc\\n123\" in first/
+        assert ex.message =~ /Cannot store newlines in combined fields storing \"abc\\n123\" in first/
```

Comment:
Again I think `assert_raises` works well. And the match can be explicitly tested:
```
assert_match /Cannot store newlines in combined fields storing \"abc\\n123\" in first/, ex.message
```
which is nicer because if it fails, we'll get a useful message instead of "false is not true".